examples/asr/conf/conformer/fastconformer/fastconformer_ctc_bpe.yaml

```diff
@@ -0,0 +1,203 @@
+# It contains the default values for training a Conformer-CTC ASR model, large size (~120M) with CTC loss and sub-word encoding.
```

Comment:
Take the configs out of conf/conformer/ and put them in conf/fastconformer/

Reply:
Treat it as an entirely separate model architecture then normal conformer.

Reply:
You need to mention fast conformer here.